core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -140,7 +140,6 @@
     public static final String APP = "app";
     public static final String QUARKUS = "quarkus";
     public static final String DEFAULT_FAST_JAR_DIRECTORY_NAME = "quarkus-app";
-    public static final String RENAMED_JAR_EXTENSION = ".jar.original";
```

Comment:
@mariofusco @evacchi i am not sure it is relevant for you but given that @mariofusco introduced this constant i wanted to check whether removing it is ok.

Reply:
I only found this reference in a comment but I honestly don't even know if it's still relevant https://github.com/kiegroup/kogito-runtimes/blob/bece507d15db8b3270d4d35408b29ec34dffc1de/kogito-quarkus-extension/deployment/src/main/java/org/kie/kogito/quarkus/deployment/AppPaths.java#L132 @mariofusco

Reply:
So it looks like it's even better. With this change it will always be `jar`.